extractors/bookseries/karelians/extractors/forest_extractor.py

```diff
@@ -0,0 +1,26 @@
+import re
+
+import core.utils.regex_utils as regexUtils
+from core.pipeline_construction.base_extractor import BaseExtractor
+
+
+class ForestExtractor(BaseExtractor):
+    extraction_key = 'metsää'
+
+    def __init__(self, cursor_location_depends_on=None, options=None):
+        super(ForestExtractor, self).__init__(cursor_location_depends_on, options)
+        self.FOREST_PATTERN = r'(?:([0-9][0-9]) ha metsää){s<=1}|(?<!H)(a metsää){s<=1}|(metsää [0-9][0-9] ha){s<=1}'
+        self.FOREST_OPTIONS = (re.UNICODE | re.IGNORECASE)
+
+    def _extract(self, entry, extraction_results, extraction_metadata):
+        own_forest = self._find_forest(entry["text"])

```

Comment:
`own_forest` doesn't fully convey to me what this variable exactly holds. I mean I can know by reading the implementation of `_find_forest`, but I should be able to know from the variable name what it is or holds. The more explicit, the better :relaxed:  If it's a flag for whether the person owns forest, it could be something like `owns_forest` or even `person_owns_forest`, or `has_forest_property`. If it's the area of the forest, maybe something along the lines of `owned_forest_area`. Or `forest_area_owned` :thinking:  Naming variables well is really challenging :'D